docs/source/conf.py

```diff
@@ -441,13 +452,67 @@ def check_timeout_and_api_kwargs_presence(obj: object) -> int:
     )
 
 
+def create_return_admonitions() -> dict[str, str]:
+    """Creates 'Returned in' admonitions for classes that are returned in Bot's methods."""
+
+    methods_for_class = defaultdict(list)
+
+    # checking for islower() to avoid camelCase methods
+    for method in [m for m in dir(telegram.Bot) if not m.startswith("_") and m.islower()]:
+        try:
+            sig = inspect.Signature.from_callable(getattr(telegram.Bot, method))
```

Comment:
```suggestion
            sig = inspect.signature(getattr(telegram.Bot, method))
```

that's actually just [a shortcut](https://github.com/python/cpython/blob/ba88628808af14ec904a9afe3b72dca9159d7ce8/Lib/inspect.py#L3277), but at least it's shorter :D

Reply:
Done